neo4j-bolt/src/test/groovy/io/micronaut/neo4j/bolt/Neo4jEmbeddedServerSpec.groovy

```diff
@@ -64,4 +67,27 @@ class Neo4jEmbeddedServerSpec extends Specification{
         cleanup:
         applicationContext?.stop()
     }
+
+    void "test neo4j testcontainer"() {
+        given:
+        Neo4jContainer neo4jContainer = new Neo4jContainer(DockerImageName.parse("neo4j:latest"))
```

Comment:
please use a tag which matches the driver version instead of `latest`

Reply:
@sdelamo there is no tag that matches driver version 5.11.0. Latest image is 4.4.24.